pkg/cmd/datastore/datastore.go

```diff
@@ -25,7 +25,10 @@ import (
 
 type engineBuilderFunc func(ctx context.Context, options Config) (datastore.Datastore, error)
 
-const MaxReplicaCount = 16
+const (
+	MaxReplicaCount          = 16
```

Comment:
Where is `MaxReplicaCount` used?

Reply:
I didn't add it (you did) but it's used when validating the replica config for a pg instance.

Reply:
Oh, sorry. I thought it was new. My bad.